USAGE_DATA.md

```diff
@@ -0,0 +1,11 @@
+## [IntelliJ Knative by Red Hat](https://github.com/redhat-developer/intellij-knative)
+
+### Usage Data
+
+* when plugin is started
+* when an action is executed
+    * action name
+    * action duration time
+    * action error message (in case of exception)
+    * action specific data if applicable (see details below)
```

Comment:
Can't find it. Do we really have action specific data. If not we should remove this